src/main/java/com/techswitch/ispy/services/ReportService.java

```diff
@@ -37,7 +37,10 @@ public ReportDatabaseModel createReport(ReportRequestModel reportRequestModel) {
     }
 
     public List<ReportDatabaseModel> getAllReports(Filter filter){
-        return jdbi.withHandle(handle -> handle.createQuery("SELECT * FROM reports LIMIT :limit OFFSET :offset")
+        return jdbi.withHandle(handle -> handle.createQuery("select id, suspect_id, " +
+                "to_char(date_of_sighting, 'dd-mm-yyyy') as date_of_sighting, location, description, " +
+                "to_char(timestamp_submitted, 'dd-mm-yyyy  HH24:mm:ss') as timestamp_submitted " +
```

Comment:
this looks a lot less pleasant than what we had before.
Sounds like our `ReportDatabaseModel` doesn't match whats actually in the database.
Much better fix would be to change that model.